.env.example

```diff
@@ -11,6 +11,12 @@ DB_DATABASE=homestead
 DB_USERNAME=homestead
 DB_PASSWORD=secret
 
+TESTING_DB_HOST=127.0.0.1
+TESTING_DB_PORT=3306
+TESTING_DB_DATABASE=homestead_testing
+TESTING_DB_USERNAME=homestead
+TESTING_DB_PASSWORD=secret
```

Comment:
Probably it's will be better to move this setting to .env.testing without TESTING_ prefix

Reply:
ok